### PR TITLE
Chronos: Split TF UT, cuz we have both tf1 and tf2

### PR DESCRIPTION
**Pull request by liangs6212**
I think this PR should be merged after `Keras` model.

### CODE REVIEW

**Comment by liangs6212** [comment]
I think this PR should be merged after `Keras` model.

File: python/chronos/dev/test/run-pytests-tf1.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    export OMP_NUM_THREADS=1
+fi
+
+ray stop -f
+
+echo "Running chronos tests TF1 and Deprecated API"
+python -m pytest -v test/bigdl/chronos/forecaster/test_mtnet_forecaster.py \
+                    test/bigdl/chronos/model/test_mtnet.py \
+                    test/bigdl/chronos/autots/deprecated/ \
+                    test/bigdl/chronos/model/tf1/
```
Comment (TheaperDeng) [inline python/chronos/dev/test/run-pytests-tf1.sh:34]: This PR will be merged ahead of #4083 , I don't think we have /model/tf1 at this time

File: python/chronos/dev/test/run-pytests.sh [hunk 2]
```diff
@@ -60,7 +60,7 @@ python -m pytest -v test/bigdl/chronos/autots\
                     test/bigdl/chronos/data \
                     test/bigdl/chronos/simulator \
                     test/bigdl/chronos/detector \
-       -k "not test_forecast_tcmf_distributed"
+       -k "not deprecated"
```
Comment (TheaperDeng) [inline python/chronos/dev/test/run-pytests.sh:63]: only `not deprecated` is enough?

File: python/chronos/dev/test/run-pytests-tf1.sh (hunk 1 above)
Comment (liangs6212) [reply python/chronos/dev/test/run-pytests-tf1.sh:34]: Ok, I will change this.

File: python/chronos/dev/test/run-pytests.sh (hunk 2 above)
Comment (liangs6212) [reply python/chronos/dev/test/run-pytests.sh:63]: I'm sure other tf1 ut will be skipped, so we just need to specify "-k not deprecated".

File: python/chronos/dev/test/run-pytests.sh [hunk 3]
```diff
@@ -60,7 +60,7 @@ python -m pytest -v test/bigdl/chronos/autots\
                     test/bigdl/chronos/data \
                     test/bigdl/chronos/simulator \
                     test/bigdl/chronos/detector \
-       -k "not test_forecast_tcmf_distributed"
+        -k "not test_ae_fit_score_unrolled"
```
Comment (TheaperDeng) [inline python/chronos/dev/test/run-pytests.sh:63]: `-k "not test_ae_fit_score_unrolled and not test_forecast_tcmf_distributed"`?

File: python/chronos/dev/test/run-pytests-tf1.sh [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+export PYSPARK_DRIVER_PYTHON=python
+if [ -z "${OMP_NUM_THREADS}" ]; then
+    export OMP_NUM_THREADS=1
+fi
+
+ray stop -f
+
+echo "Running chronos tests TF1 and Deprecated API"
+python -m pytest -v test/bigdl/chronos/forecaster/test_mtnet_forecaster.py \
+                    test/bigdl/chronos/model/test_mtnet.py \
```
Comment (TheaperDeng) [inline python/chronos/dev/test/run-pytests-tf1.sh:32]: test/bigdl/chronos/model/test_Seq2Seq.py
test/bigdl/chronos/model/test_VanillaLSTM.py

File: python/chronos/dev/test/run-pytests.sh (hunk 3 above)
Comment (liangs6212) [reply python/chronos/dev/test/run-pytests.sh:63]: In fact, `test_forecast_tcmf_distributed` exists only in UT part1.

Comment (TheaperDeng) [reply python/chronos/dev/test/run-pytests.sh:63]: I see, thx

**Review by TheaperDeng — APPROVED** [review]
LGTM, we will discuss jenkins job config later

**Comment by TheaperDeng** [comment]
As we have discussed, please only test tf1 for python3.6 and tf2 for python3.7

**Comment by liangs6212** [comment]
> As we have discussed, please only test tf1 for python3.6 and tf2 for python3.7

Passed, related url: http://10.112.231.51:18889/job/BigDL-Chronos-PR-Validation-Env-Test/23/

**Comment by TheaperDeng** [comment]
http://10.112.231.51:18888/job/BigDL-Chronos-PR-Validation/474/

**Pull request merged**